bokeh/io.py

```diff
@@ -612,32 +615,35 @@ def gridplot(plot_arrangement, **kwargs):
     _push_or_save(grid)
     return grid
 
+@deprecated("Bokeh 0.12.0", "bokeh.models.layouts.Row")
 def hplot(*children, **kwargs):
-    ''' Generate a layout that arranges several subplots horizontally.
-
-    '''
-    _remove_roots(children)
-    layout = HBox(children=list(children), **kwargs)
-    curdoc().add_root(layout)
-    _push_or_save(layout)
+    warnings.warn(
+        """
+        The new Row is responsive by default, it resizes based on the space available. If you would
+        like to keep using a fixed size row like hplot you can set responsive=False
+        on Row. This has been automatically set on hplot.
+        """)
+    layout = Row(children=list(children), responsive=False, **kwargs)
```

Comment:
I guess I'm not communicating very well!!!

Row is responsive True. To reduce surprise layouts I set `responsive=False` when creating a Row for someone using hplot, etc. What I was trying to say is:
1) hplot is deprecated
2) Row has different defaults
3) if you want to use Row but have it behave like hplot then set `responsive=False`
4) this has been done for you in hplot